tests/integration/main.go

```diff
@@ -38,6 +38,11 @@ func main() {
 	dmc.ValidateStringLiterals()
 	dmc.startDocker()
 
+	slash := SlashThrottleTestRun()
```

Comment:
Is there an issue tracking what tests are run for CI vs what has to be ran manually? Maybe we add this to the QA doc, or some other source of truth until https://github.com/cosmos/interchain-security/issues/514 is completed

Reply:
Ideally everything will run in the CI soon, I would not open new issues.